ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/stun.schema.yml

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) 2023 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+# yaml-language-server: $schema=../../../../plugins/plugin_utils/schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  stun:
+    type: dict
+    keys:
+      client:
+        type: dict
+        description: |
+          "If a device is a client"
+        keys:
+          server_profiles:
+            type: list
+            description: |
+              Enable Server profiles need to reference the name of the profile and IP address
+              Examples:
+              - "server-profile profile1
```

Comment:
Is this example based on the CLI or how a yaml example would look like? Also, please remove extra quotes